prometheus/getmessages_prometheus.py

```diff
@@ -267,10 +395,10 @@ def get_if_config_vars():
         for p_type in project_type:
             if p_type not in {
                     'METRIC',
-                    #'LOG',                 ## enable these when addressing II-5158
-                    #'INCIDENT',
-                    #'ALERT',
-                    #'DEPLOYMENT'
+                    'LOG',                 
```

Comment:
Is it possible to send these data types through this agent?

Reply:
Yep, I added Alerts in this update as well

Reply:
Great thanks